### PR TITLE
[COMSUP-49] Document `watch`  function for prefetching

### PR DESCRIPTION
**Pull request by tristanlee85**

### CODE REVIEW

File: src/guides/v7/performance/prefetching.md [hunk 1]
```diff
@@ -46,27 +34,57 @@ After you have created a service worker, your router will need to be configued t
...
+  })
+  // serve the service worker from the /dist directory
+  .match('/service-worker.js', ({serviceWorker}) => {
+    serviceWorker('dist/service-worker.js');
+  });
+```
+
+## Registering the Service Worker {/* registering-the-service-worker */}
+
+Once you've created and defined a route to serve the service worker, code running in the browser window needs to register the service worker before prefetching can begin. How you do this depends on the front-end framework that you use.
```
Comment (jsernaque) [inline src/guides/v7/performance/prefetching.md:52]: Suggestion for a future enhancement: Add more info or links for our tier 1 frameworks.

File: src/guides/v7/performance/prefetching.md [hunk 2]
```diff
@@ -46,27 +34,57 @@ After you have created a service worker, your router will need to be configued t
...
+    ],
   });
+});
 ```
 
+Now when your client-side code runs, the service worker will be installed and will begin prefetching URLs.
+
 ## Configuring Routes for Prefetching {/* configuring-routes-for-prefetching */}
 
 To ensure that excessive prefetch traffic isn't passed on to your origin, {{ PRODUCT_NAME }} will serve prefetch requests when a cached response is available at the edge. By default, all prefetch requests will be cached at the edge for 2 minutes (see [`DEFAULT_MAX_AGE_SECONDS`](/docs/api/prefetch/interfaces/sw_Prefetcher.PrefetcherConfig.html#defaultMaxAgeSeconds)). Additionally, you may configure a route that caches responses at the edge and in the service worker within your router, optionally giving it longer cache time for greater performance. In this example we define a route that caches product API calls for one hour:
```
Comment (jsernaque) [inline src/guides/v7/performance/prefetching.md:90]: We should verify that the default caching policy for prefetching is still 2 minutes.

File: src/guides/v7/performance/prefetching.md [hunk 3]
```diff
@@ -46,27 +34,57 @@ After you have created a service worker, your router will need to be configued t
...
+        },
+      },
+    ],
   });
+});
 ```
 
+Now when your client-side code runs, the service worker will be installed and will begin prefetching URLs.
+
 ## Configuring Routes for Prefetching {/* configuring-routes-for-prefetching */}
```
Comment (jsernaque) [inline src/guides/v7/performance/prefetching.md:88]: What if we changed this to something more specific? 
Example: Defining a Prefetching Caching Policy

File: src/guides/v7/performance/prefetching.md (hunk 2 above)
Comment (tristanlee85) [reply src/guides/v7/performance/prefetching.md:90]: It's not exported in the API docs, but this is in the source:

```
/**
 * The amount of time prefetch requests are cached for when no explicit TTL is set in the route definition or
 * query param.
 */
export const DEFAULT_MAX_AGE_SECONDS = 60 * 2 // 2 minutes
```

**Review by jsernaque — APPROVED** [review]

**Pull request merged**